### ISSUE TITLE
Please make confirmation email "from" address configurable

### ISSUE THREAD

**Issue by Toutouwai** [ryancramerdesign/LoginRegister#5]
LoginRegister sends the confirmation email with no explicit "from" address, meaning WireMail falls back to the $config->adminEmail address when one is defined.

But the adminEmail address is for receiving error notices, so it is usually the address of the developer - this is not the address that confirmation emails should be coming from. Could you please add a config field for the module to define the default address, like what is available in the Forgot Password module? Thanks.

I don't think WireMail should be falling back to $config->adminEmail at all, for the same reason: it's usually not the developer who should be perceived as the sender of emails, but should be some generic address associated with the website domain. So I'll open a [feature request](https://github.com/processwire/processwire-requests/issues/122) separately to have a configurable default "from" address for WireMail also.

**Comment by abdusco** [comment]
While I agree with everything you said, (correct me if I'm missing anything) exposing a setting on the admin side is only available through module configurations. There isn't any control center where we can change these settings. This would mean either:

1. Developing a **General Settings** functionality, like a front for $config, but for non-critical settings (which would be great 😃 )
2. Turning WireMail (a core class) into a module (which I doubt @ryancramerdesign will agree with)

**Comment by Toutouwai** [comment]
@abdusco, I don't follow you here - maybe you meant to comment on the other issue (WireMail feature request)?

LoginRegister is already a configurable module - it just needs a field for "Confirmation email 'from' address" and to use that in `$mail->from()`.

As for making the default WireMail from address configurable, this can just be a setting in config.php, e.g. `$config->defaultFrom`

**Comment by abdusco** [comment]
> @abdusco, I don't follow you here - maybe you meant to comment on the other issue (WireMail feature request)?

Yeah, I noticed it after posting the comment, but didn't want to duplicate the comment there.

> As for making the default WireMail from address configurable, this can just be a setting in config.php, e.g. $config->defaultFrom

I was thinking of a settings page that only users with a certain permissions can use. Developers can expose certain settings for modification, similar to ModuleConfig class, but without any ties to page structure, and settings stored as JSON, for example. This way we don't have to create trivial, single-use fields for checkboxes, textboxes etc.

But yours is much easier.

**Comment by Toutouwai** [comment]
> Developing a General Settings functionality, like a front for $config, but for non-critical settings (which would be great 😃 )

Do you mean you would want to give non-superusers access to this? Because for superusers there is already the [ProcessWireConfig](https://github.com/ryancramerdesign/ProcessWireConfig) module.
